bin/bot.ts

```diff
@@ -114,7 +114,7 @@ const config: BotConfig = {
   orderAlgoDepth,
   api,
 };
-Object.freeze(config);
+// Object.freeze(config);
```

Comment:
I was wondering why the config is no longer being frozen? This was being done just for a state safety reason (isn't necessary tho).

Reply:
Yea, I know right but I had to unfreeze it because I needed to set the poolInfo Address inside the config

Reply:
The config is passed in from the GUI to the lib code and that's why it's called input config. The pool info shouldn't be part of this config IMO since it's not input.